docs/google/engine.md

```diff
@@ -129,6 +129,14 @@ has a method `run_batch()` that functions similar to `run()` but accepts a
 list of circuits and parameter sweeps.  Each circuit must have a corresponding
 parameter sweep.  If the circuit does not use a sweep, pass in `None`.
 
+There are some restrictions on the circuits that can be batched together.
+All circuits in the same batch must measure the same set of qubits.  They
+must all contain the same number of repetitions.
+Batching circuits together that do not follow this restriction may not
```

Comment:
"may not" or "will not"

Reply:
Well, I put "may not" since I am not totally sure how the batch sampler stuff will land.  If you specify different repetitions in the Sampler interface, will it raise an exception or silently let it slide?   Right now it is still TBD.